packages/next/build/webpack-config.ts

```diff
@@ -936,6 +936,9 @@ export default async function getBaseWebpackConfig(
     include: [dir, /next[\\/]dist[\\/]pages/],
   }
 
+  const rscSharedRegex =
+    /(node_modules\/react\/|\/shared\/lib\/(head-manager-context|router-context)\.js|node_modules\/styled-jsx\/)/
```

Comment:
```suggestion
    /(node_modules\/react\/|\/shared\/lib\/(head-manager-context|router-context|app-router-context)\.js|node_modules\/styled-jsx\/)/
```
Do we want the new app router context here as well?

Reply:
app-router-context is not been used on the server side yet, I think it's good so far. cc @shuding can you confirm?

Reply:
We can include it there, eventually it will be used there

Reply:
got it, let's add it later